ui/e2e/src/tests/timePicker.spec.ts

```diff
@@ -0,0 +1,121 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { test, expect } from '../fixtures/dashboardTest';
+
+test.use({
+  dashboardName: 'Panels',
+
+  // Friday, February 10, 2023 12:00:15 PM GMT-08:00
+  mockNow: 1676059215000,
+});
+
+test.describe('Time Picker', () => {
```

Comment:
I wrote e2e tests instead of unit tests because I ran into a bunch of issues trying to test the clock face interactions with RTL.

Reply:
One more reason to get rid of the MUI clock completely, can't wait to see Mohit's ideas for the next iteration